packages/validator/src/services/indices.ts

```diff
@@ -110,6 +116,49 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
```

Comment:
We usually sort modules first, arguments second. May be good to add to contributing guidelines

Reply:
not sure what you mean by this

Reply:
can just make `logValidatorStatuses` a private method of the class then we do not have to worry about the order of function parameters here. Having `allValidators` as persistent as private property of the class might also make it easier to implement https://github.com/ChainSafe/lodestar/issues/5158. Though could refactor that later once we know how to add the metric for this.

Reply:
What's the significance of the order of the function parameters? The reason why I didn't make it a class method was because I don't think it relates directly to the Indices service but rather it is a helper function.

Reply:
> The reason I didn't make it a class method was because I don't think it relates directly to the Indices service but rather it is a helper function.

IMO if the function is just used in this class would go for a private method. Also isn't one of the jobs of the indices service to print out information about the validators? Either way is fine, just personal preference